convert/jenkinsjson/json/s3upload.go

```diff
@@ -0,0 +1,103 @@
+package json
+
+import (
+	"fmt"
+
+	harness "github.com/drone/spec/dist/go"
+)
+
+func ExtractEntries(node Node) []map[string]interface{} {
```

Comment:
Provide a quick doc on why that method is required.

Reply:
ok